src/commands/trialApp/removeTrialApp.ts

```diff
@@ -13,7 +14,13 @@ export async function removeTrialApp(context: IActionContext, node?: TrialAppTre
         node = await ext.tree.showTreeItemPicker<TrialAppTreeItem>(TrialAppTreeItem.contextValue, context);
     }
 
-    ext.context.globalState.update(TrialAppLoginSession, undefined);
-    delete ext.azureAccountTreeItem.trialAppNode;
-    await ext.tree.refresh();
+    const message: string = `Are you sure you want to remove Trial app "${node.client.fullName}"?`;
```

Comment:
nit: We should localize any new string going forward.  

Also, don't like the casing.  We should either do Trial App or trial app.  lol I just noticed you're pretty inconsistent in your tutorial about this but you have generally been using "trial apps" in messages.